proptest/CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 ## Unreleased
 
+## 1.2.0
```

Comment:
did a quick scan over the commits since 1.1 to see if we're missing anything in changelog. seems like we dont have anything for:
- we're now publishing MSRV
- remove `quick_error` dependency

I can add a commit adding these shortly.

side note: just realized we had some changes to the derive crate but dont think we've published that yet or updated its changelog:
- updated `syn`, `quote` and `proc_macro2` to

Reply:
latest commit updates changelogs for both proptest and proptest-derive